pkg/hints/math_hints_test.go

```diff
@@ -162,6 +165,80 @@ func TestAssertNotZeroHintFail(t *testing.T) {
 	}
 }
 
+func TestVerifyValidSignature(t *testing.T) {
+	vm := NewVirtualMachine()
+	vm.Segments.AddSegment()
+	signature_builtin := builtins.NewSignatureBuiltinRunner(2048)
+	vm.BuiltinRunners = append(vm.BuiltinRunners, signature_builtin)
+
+	hintProcessor := CairoVmHintProcessor{}
+	vm.Segments.AddSegment()
+
+	r_felt := lambdaworks.FeltFromDecString("3086480810278599376317923499561306189851900463386393948998357832163236918254")
+	s_felt := lambdaworks.FeltFromDecString("598673427589502599949712887611119751108407514580626464031881322743364689811")
+	r := memory.NewMaybeRelocatableFelt(r_felt)
+	s := memory.NewMaybeRelocatableFelt(s_felt)
+
+	vm.RunContext.Fp = memory.NewRelocatable(1, 3)
+
+	idsManager := SetupIdsForTest(
+		map[string][]*MaybeRelocatable{
+			"ecdsa_ptr":   {nil},
+			"signature_r": {r},
+			"signature_s": {s},
+		},
+		vm,
+	)
+
+	hintData := any(HintData{
+		Ids:  idsManager,
+		Code: VERIFY_ECDSA_SIGNATURE,
+	})
+
+	err := hintProcessor.ExecuteHint(vm, &hintData, nil, nil)
+
+	if err != nil {
+		t.Errorf("Verify signature hint for correct signature failed with error: %s", err)
+	}
+}
+
+func VerifySignatureInvalidEcdsaPointer(t *testing.T) {
```

Comment:
This test is not being run as it does not start with `Test`

Reply:
Fixed